main.nf

```diff
@@ -360,6 +348,12 @@ process variant_effect_predictor {
               exit 1;;
     esac
 
+    # VEP failes files without variants, but the pipeline should still run
```

Comment:
Nitpick: Shell variables should be double quoted. `"\$INFILE"`, etc.

Lines 353--374